index.bs

```diff
@@ -11113,48 +11108,85 @@ is the concatenation of the [=interface=]’s
     making \[[PreventExtensions]] fail.
 </div>
 
+
 <h4 id="es-constants">Constants</h4>
 
-For each [=exposed=] [=constant=] defined on an [=interface=] |A|,
-there must be a corresponding property.
-The property has the following characteristics:
+[=Constants=] are exposed on [=interface objects=],
+[=legacy callback interface objects=],
+[=interface prototype objects=], and
+on the single object that implements the interface,
+when an interface is declared with the [{{Global}}] [=extended attribute=].
 
-*   The name of the property is the [=identifier=] of the [=constant=].
-*   The location of the property is determined as follows:
-    *   If the interface was declared with the [{{Global}}] [=extended attribute=],
-        then the property exists on the single object that implements the interface.
-    *   Otherwise, if the interface has an [=interface prototype object=],
-        then the property exists on it.
-*   The value of the property is that which is obtained by [=converted to an ECMAScript value|converting=] the [=constant=]’s IDL value to an ECMAScript value.
-*   The property has attributes { \[[Writable]]: <emu-val>false</emu-val>, \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: <emu-val>false</emu-val> }.
+<div algorithm>
+    To <dfn>define the constants</dfn> of [=interface=] or [=namespace=] |definition| on |target|,
```

Comment:
Namespaces can't have constants, so no need for that.

Reply:
Fixed in https://github.com/heycam/webidl/pull/494/commits/6d426d8fb24389a241e25b5f5ada0af35b890fe0.